pnpm-lock.yaml

```diff
@@ -16837,6 +16689,30 @@ packages:
       - supports-color
     dev: true
 
+  /@fluidframework/aqueduct/1.4.0:
+    resolution: {integrity: sha512-b3I3fWGAWuXQbyuEFeeEi3GVVUOYPWJfFaB4httsu5Jn4C0QSkKxftkielDlor9/7rCJYjHc4IEWViaVO+kBbA==}
+    dependencies:
+      '@fluidframework/common-definitions': 0.20.1
+      '@fluidframework/common-utils': 0.32.2
+      '@fluidframework/container-definitions': 1.4.0
+      '@fluidframework/container-loader': 1.4.0
+      '@fluidframework/container-runtime': 1.4.0
+      '@fluidframework/container-runtime-definitions': 1.4.0
+      '@fluidframework/core-interfaces': 1.4.0
+      '@fluidframework/datastore': 1.4.0
+      '@fluidframework/datastore-definitions': 1.4.0
+      '@fluidframework/map': 1.4.0
+      '@fluidframework/request-handler': 1.4.0
+      '@fluidframework/runtime-definitions': 1.4.0
+      '@fluidframework/runtime-utils': 1.4.0
+      '@fluidframework/synthesize': 1.4.0
+      '@fluidframework/view-interfaces': 1.4.0
+      uuid: 8.3.2
+    transitivePeerDependencies:
+      - debug
+      - supports-color
+    dev: false
```

Comment:
This and a number of other additional dependencies are listed as non-dev dependencies.
Non-dev dependencies on older packages make it hard to determine what if anything is creeping into production. This includes good Component Governance monitoring.
Can all of these additions be changed to be dev dependencies?
cc: @ChumpChief